Yellow Light

DrOrnage opened the pull request:
Addition of yellow light to indicate the wrong input of users while doing the binary checking.

DrOrnage committed:
Yellow Light

Addition of yellow light to indicate the wrong input of users while doing the binary checking.

dadkofrajer reviewed (changes requested):
You need to resolve the conflicts as your code is merging into the current existing code

DrOrnage committed:
Yellow light

Addition of yellow lights to indicate wrong input of users during the check function.

DrOrnage committed:
Yellow light

Addition of yellow lights to indicate wrong input of users during the check function.

DrOrnage committed:
Merge branch 'main' into base2_light

dadkofrajer reviewed (approved):
Yep looks good.
